generator/osm_source.cpp

```diff
@@ -340,6 +341,15 @@ class MainFeaturesEmitter : public EmitterBase
 
     if (info.m_createWorld)
       m_world.reset(new TWorldGenerator(info));
+
+    string const featureId2OsmIdsFile = info.GetIntermediateFileName(info.m_featureIdToOsmIds, "");
+    LOG(LINFO, ("Saving mapping from feature id osm ids to file", featureId2OsmIdsFile));
```

Comment:
Saving mapping from feature ids to osm ids to
OR
Saving the (feature id -> osm ids) mapping to